bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/HDPowerViewWebTargets.java

```diff
@@ -226,6 +228,9 @@ private ShadeData shadeDataFromJson(String json) throws HubInvalidResponseExcept
             if (shadeData == null) {
                 throw new HubInvalidResponseException("Missing 'shade.shade' element");
             }
+            if (Boolean.TRUE.equals(shadeData.timedOut)) {
```

Comment:
@jlaur a question: does the hub really return a JSON response containing a `timedout` element? I never observed it, but it is interesting.

Reply:
Yes, it does. For me it happens quite often when hard-refreshing positions. In general, my experience is that the radio implementation is not very reliable.

Reply:
I have 11 shades... this is my most recent log history:
```
2022-02-03 19:35:00.856 [INFO ] [rnal.handler.HDPowerViewShadeHandler] - Shade 23137 wireless refresh time out
2022-02-03 19:50:05.650 [INFO ] [rnal.handler.HDPowerViewShadeHandler] - Shade 24076 wireless refresh time out
2022-02-03 20:04:51.436 [INFO ] [rnal.handler.HDPowerViewShadeHandler] - Shade 23137 wireless refresh time out
2022-02-03 20:05:17.946 [INFO ] [rnal.handler.HDPowerViewShadeHandler] - Shade 24076 wireless refresh time out

```

Reply:
Ok. Interesting. I only have 3 shades.